bundler-packages/resolutions-rs/wasi-worker-browser.mjs

```
@@ -1,32 +1,32 @@
-          ...importObject.napi,
-          ...importObject.emnapi,
-          memory: wasmMemory,
-        }
-      },
-    })
-  },
+	onLoad({ wasmModule, wasmMemory }) {
+		const wasi = new WASI({
+			print: () => {
+				// eslint-disable-next-line no-console
+				console.log.apply(console, arguments)
```

Biome (1.9.4)

[error] Use the rest parameters instead of arguments.

arguments does not have Array.prototype methods and can be inconvenient to use.

(lint/style/noArguments)